rmw_implementation/rmw_implementation-extras.cmake

```diff
@@ -14,10 +14,7 @@
 
 # copied from rmw_implementation/rmw_implementation-extras.cmake
 
-include(
-  "${rmw_implementation_DIR}/add_executable_for_each_rmw_implementation.cmake")
-include("${rmw_implementation_DIR}/get_available_rmw_implementations.cmake")
-include("${rmw_implementation_DIR}/get_default_rmw_implementation.cmake")
+find_package(rmw_implementation_cmake REQUIRED)
```

Comment:
The package doesn't seem to have an exec dependency on `rmw_implementation_cmake` right now?